package/piksi_system_daemon/src/Makefile

```diff
@@ -1,5 +1,6 @@
 TARGET=piksi_system_daemon
 SOURCES= \
+	async-child.c \
```

Comment:
It doesn't matter to me. I've just seen nothing but underscores for the longest time.